manifests/w/warpdesign/react-explorer/2.3.1/warpdesign.react-explorer.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: warpdesign.react-explorer

```

```suggestion
PackageIdentifier: NicolasRamz.React-Explorer
```
I suggest using installer provided names instead of github names.